packages/wallet/src/services/ProviderTracker/HandleProvider.ts

```diff
@@ -0,0 +1,45 @@
+import { HandleProvider, HandleResolution, ResolveHandlesArgs } from '@cardano-sdk/core';
+import { HealthCheckResponse, NetworkInfoProvider, ProviderError, ProviderFailure } from '@cardano-sdk/core/src';
+import { IHandle } from '@koralabs/handles-public-api-interfaces';
+import { toHandleInfo } from '../util';
+import axios, { AxiosInstance } from 'axios';
+
+export interface KoraLabsHandleProviderDeps {
+  serverUrl: string;
+  networkInfoProvider: NetworkInfoProvider;
```

Comment:
This will probably be needed to make it work in the browser (see `cardano-services-client` implementations)
```suggestion
  networkInfoProvider: NetworkInfoProvider;
  axiosAdapter: AxiosAdapter;
```